libres/lib/analysis/analysis_module.cpp

```diff
@@ -100,26 +101,23 @@ static UTIL_SAFE_CAST_FUNCTION(analysis_module, ANALYSIS_MODULE_TYPE_ID)
 
 void analysis_module_initX(analysis_module_type *module, matrix_type *X,
```

Comment:
We could also remove these functions as they are only used a test

Reply:
Yes!

Reply:
Done